modules/library.js

```diff
@@ -0,0 +1,44 @@
+import loadBooks from './loadBooks.js'; // eslint-disable-line import/no-cycle
```

Comment:
- [ ] @fdezcaminero I have seen you have disables the linter check in some lines, I would strongly recommend you try and fix the errors before disabling the check.